common/libraries/clients/postgresql/src/main/java/org/eclipse/dataspaceconnector/clients/postgresql/statement/NullArgumentHandler.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Copyright (c) 2021 Daimler TSS GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Daimler TSS GmbH - Initial API and Implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.clients.postgresql.statement;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public enum NullArgumentHandler implements ArgumentHandler {
+    INSTANCE;
```

Comment:
Is an enum containing one element necessary?

Reply:
As this is also discussed in PR #439, where this code was introduced and this PR is based on, I would resolve the comment here. Like this we have the discussion only in one place. I hope this is ok.